index.go

```diff
@@ -561,7 +561,47 @@ func (i *Index) recalculateCaches() {
 	}
 }
 
-// CreateField creates a field.
+// createNullableField is just like CreateField, except that it allows
```

Comment:
I am not 100% sure how we can get here. The thing I'm trying to deal with is roughly the case where we're processing schema stuff that was already processed somewhere else, such as bringing up a node in a cluster that has a field this node somehow failed to previously hear about, and I don't want the node to do the wrong thing there. So it's not specifically about serverless, but honestly I'm not totally sure which paths we end up on where.

My testing for this stuff basically consisted of running old featurebase, creating a field, then running new featurebase and verifying that the field reported empty sets instead of nulls in that field, and rejected attempts to query the field for is/is-not null.